test/test-makefile.sh

```diff
@@ -137,8 +137,22 @@ get_device_guid () {
 
 run_fido_device_onboard () {
   local log=$1
+  local additional_params=$2
   cd ${creds_dir}
-  go-fdo-client --blob "${device_credentials}" --debug onboard --key ec256 --kex ECDH256 | tee "${log}"
+  
```

Comment:
I hit this as well, need to pass additional client args: https://github.com/fido-device-onboard/go-fdo-server/pull/67/files#diff-8c08654a361e08e0446fee0e519021147430ee892513137363660a84c944ce51R140

But instead of putting the FSIM specific arguments in this function I pass those along from the wget test file itself: https://github.com/fido-device-onboard/go-fdo-server/pull/67/files#diff-eb3a45980a25f9ca4c3cf84f2a4b264b616e3a53ddb2a4be2027a09620227afcR156